metadata.json

```diff
@@ -1,10 +1,10 @@
 {
-"shell-version": ["3.18", "3.20", "3.22", "3.24", "3.26", "3.28", "3.30"],
+"shell-version": ["3.31", "3.32"],
 "uuid": "dash-to-dock@micxgx.gmail.com",
 "name": "Dash to Dock",
 "description": "A dock for the Gnome Shell. This extension moves the dash out of the overview transforming it in a dock for an easier launching of applications and a faster switching between windows and desktops. Side and bottom placement options are available.",
 "original-author": "micxgx@gmail.com",
 "url": "https://micheleg.github.io/dash-to-dock/",
 "gettext-domain": "dashtodock",
-"version": 64
+"version": 65
```

Comment:
This was probably not intended.

Reply:
Well I think we need a release bump after this, but probably should be put in a different commit I guess, right?

Reply:
Yes, I usually prefer to bump the version at release.

Reply:
Ok, done